sunpy/net/dataretriever/sources/rhessi.py

```diff
@@ -207,3 +207,10 @@ def _can_handle_query(cls, *query):
                 matches = False
 
         return matches
+
+    @classmethod
+    def register_values(cls):
+        from sunpy.net import attrs
+        adict = {attrs.Instrument: [('RHESSI',
```

Comment:
This client now also provides `a.Physobs("summary_lightcurve")`

Reply:
Fixed.